From 966091c79baa04b53a9790c3cd26ebfd49c3aa68 Mon Sep 17 00:00:00 2001
From: AMS21 <AMS21.github@gmail.com>
Date: Sat, 11 May 2024 12:00:58 +0200
Subject: [PATCH] Remove usage of lseek64

---
 llvm/lib/Support/raw_ostream.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/llvm/lib/Support/raw_ostream.cpp b/llvm/lib/Support/raw_ostream.cpp
index 038ad00bd608..921ab8409008 100644
--- a/llvm/lib/Support/raw_ostream.cpp
+++ b/llvm/lib/Support/raw_ostream.cpp
@@ -677,8 +677,6 @@ uint64_t raw_fd_ostream::seek(uint64_t off) {
   flush();
 #ifdef _WIN32
   pos = ::_lseeki64(FD, off, SEEK_SET);
-#elif defined(HAVE_LSEEK64)
-  pos = ::lseek64(FD, off, SEEK_SET);
 #else
   pos = ::lseek(FD, off, SEEK_SET);
 #endif
-- 
2.44.0

